File: Chris/2022-06-08-ContainsDuplicate.cs
public class Solution {
    public bool ContainsDuplicate(int[] nums) {
        Dictionary<int, bool> numDict = new Dictionary<int, bool>();
        
        foreach(int num in nums)
        {
            if(numDict.ContainsKey(num))
            {
                return true;
            }
            
            numDict.Add(num, true);
            
        }
        
        return false;
    }
}

File: Chris/2022-06-08-ContainsDuplicate.py
class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        numDict = {}
        
        for num in nums:
            if(numDict.get(num)):
                return True
        
            numDict[num] = True
        
        return False
        # Time Comp. : O(n)
        # Space Comp : O(n)